### PR TITLE
new logging notes

### PR DESCRIPTION
**Pull request by gemmellr**

### CODE REVIEW

**Comment by gemmellr** [comment]
@clebertsuconic 

I had a quick look at your latest changes and noted some things, thought it might be easier to give them with just a draft PR.

**Comment by clebertsuconic** [comment]
@gemmellr why did you close it?

File: artemis-log-processor/src/main/java/org/apache/activemq/artemis/logprocessor/LogProcessor.java [hunk 1]
```diff
@@ -37,8 +39,9 @@
 import org.apache.activemq.artemis.logprocessor.annotation.Message;
 
 @SupportedAnnotationTypes({"org.apache.activemq.artemis.logprocessor.annotation.ArtemisBundle"})
+@SupportedSourceVersion(SourceVersion.RELEASE_11)
 public class LogProcessor extends AbstractProcessor {
-
+   // Would be good if this was in its own module seperate from the annotations so that it isnt picked up by and thus run (to no effect) by users, which it currently would be but ideally would not be.
```
Comment (clebertsuconic) [inline artemis-log-processor/src/main/java/org/apache/activemq/artemis/logprocessor/LogProcessor.java:44]: we would need two modules... I can't rely on commons for this as that would be a circular dependency.


they are retention = SOURCE.. I don't think we need to spread the jar on runtime.

**Comment by gemmellr** [comment]
> @gemmellr why did you close it?

Just as I wasnt expecting you to use the commit directly,so I figured once you knew about it I could close it hehe

**Comment by clebertsuconic** [comment]
I'm lazy... easier to merge it.. and pull the comments, and ammend on the code...


thanks for those

File: artemis-log-processor/src/main/java/org/apache/activemq/artemis/logprocessor/LogProcessor.java (hunk 1 above)
Comment (gemmellr) [reply artemis-log-processor/src/main/java/org/apache/activemq/artemis/logprocessor/LogProcessor.java:44]: Yeah, it definitely wont be needed for runtime as-is, though I thought the annotations may be needed for certain build time things that use the source ( that are not necessarily building in-tree). Perhaps sufficient just to wait and see on that though.

On checking, it seems the existing annotations are not made available so it wouldnt be a change to continue that.

**Pull request merged**